ui/src/app/pages/homepage/registration-dashboard.tsx

```diff
@@ -16,19 +16,19 @@ import {AccessModule, Profile} from 'generated/fetch';
 
 const styles = reactStyles({
   registrationPage: {
-    display: 'flex', flexDirection: 'column', paddingTop: '3%', paddingLeft: '3%',
+    display: 'flex', flexDirection: 'column',
```

Comment:
Instead of adding these, should this be turned into a `FlexColumn`?

Reply:
#wasntme
probably a good idea though

Reply:
Its in green on this PR so I reviewed it :D